modules/oe_whitelabel_search/tests/src/Kernel/SearchBlockTest.php

```diff
@@ -82,9 +82,46 @@ protected function setUp(): void {
   }
 
   /**
-   * Tests the rendering of the whitelabel search block.
+   * Tests the rendering of the whitelabel search block navigation_right region.
    */
-  public function testBlockRendering(): void {
+  public function testNavigationRightSearchBlockRendering(): void {
+    $block_entity_storage = $this->container
+      ->get('entity_type.manager')
+      ->getStorage('block');
+    $entity = $block_entity_storage->load('oe_whitelabel_search_form');
+    $builder = \Drupal::entityTypeManager()->getViewBuilder('block');
+    $build = $builder->view($entity, 'block');
+    $render = $this->container->get('renderer')->renderRoot($build);
+    $crawler = new Crawler($render->__toString());
+
+    // Assert the form rendering.
+    $form = $crawler->filter('form');
```

Comment:
Shouldn't we keep the version from 1.x here?
```suggestion
    // Select the search form in the block.
    // The block template removes the block wrapper, so the form is the root
    // element.
    $form = $crawler->filter('body > form#oe-whitelabel-search-form');
```
This is from OEL-1543.